parmed/rdkit/rdkit.py

```diff
@@ -0,0 +1,43 @@
+"""
+This package contains classes responsible for loading rdkit objects
+"""
+
+from __future__ import print_function
+from parmed.formats import PDBFile
+from parmed.utils.six.moves import StringIO
+
+try:
+    from rdkit import Chem
+except ImportError:
+    Chem = None
```

Comment:
Get rid of these 4 lines and add `from rdkit import Chem` into the function(s) where `Chem` is actually used.  This way you get an `ImportError` with a sensible error message rather than an `AttributeError` complaining about `NoneType` not having a `MolToPDBBlock` or `MolFromSmiles` attribute.  That's what I've started doing.